Installation/cmake/modules/CGAL_SetupCGALDependencies.cmake

```diff
@@ -120,7 +120,7 @@ function(CGAL_setup_CGAL_dependencies target)
     if(WIN32)
       target_compile_options(${target} ${keyword} "/fp:strict")
     else()
-      target_compile_options(${target} ${keyword} "-fp-model strict")
+      target_compile_options(${target} ${keyword} "-fp-model" "strict")
```

Comment:
@afabri: That was the issue causing the errors with the Intel Compiler. Because I had added the option `"-fp-model strict"` in one argument, instead of `-fp-model strict` in two arguments, then the FPU model was not set, and the CGAL filters were wrong.

I have remerged #2592 about the faster usage of MPFR.